identity/src/main/java/com/stripe/android/identity/camera/IdentityScanFlow.kt

```diff
@@ -73,21 +69,26 @@ internal class IdentityScanFlow(
         viewFinder: Rect,
         lifecycleOwner: LifecycleOwner,
         coroutineScope: CoroutineScope,
-        parameters: Int
+        parameters: IdentityScanState.ScanType
     ) {
         coroutineScope.launch {
             if (canceled) {
                 return@launch
             }
-            aggregator.bindToLifecycle(lifecycleOwner)
+            aggregator = IDDetectorAggregator(
+                parameters,
+                aggregateResultListener
+            )
+
+            requireNotNull(aggregator).bindToLifecycle(lifecycleOwner)
 
             analyzerPool = AnalyzerPool.of(
                 IDDetectorAnalyzer.Factory(context)
             )
 
             loop = ProcessBoundAnalyzerLoop(
                 analyzerPool = requireNotNull(analyzerPool),
-                resultHandler = aggregator,
+                resultHandler = requireNotNull(aggregator),
```

Comment:
Something i've done in the past to avoid having to use `!!` or `requireNotNull` is to apply the aggregator after creating it. Something like this:

```
aggregator = IDDetectorAggregator(
    parameters,
    aggregateResultListener
).apply { aggregator ->
    // in this context, aggregator is not nullable
}
```